tests/Behat/Resources/suites/admin/managing_invoices.yml

```diff
@@ -62,9 +63,57 @@ default:
             filters:
                 tags: "@managing_invoices && @ui"
 
+        managing_invoices_application:
+            contexts:
+                - sylius.behat.context.hook.doctrine_orm
+                - Tests\Sylius\InvoicingPlugin\Behat\Context\Hook\InvoicesContext
+
+                - sylius.behat.context.setup.channel
+                - sylius.behat.context.setup.currency
+                - sylius.behat.context.setup.customer
+                - sylius.behat.context.setup.geographical
+                - sylius.behat.context.setup.order
+                - sylius.behat.context.setup.payment
+                - sylius.behat.context.setup.product
+                - sylius.behat.context.setup.product_taxon
+                - sylius.behat.context.setup.promotion
+                - sylius.behat.context.setup.admin_security
```

Comment:
Do we really need all of these contexts?

Reply:
Probably I could remove some of them 👌